pythonforandroid/recipes/python3/patches/pyconfig_detection.patch

```diff
@@ -1,13 +1,12 @@
-diff -Nru Python-3.8.2/Lib/site.py Python-3.8.2-new/Lib/site.py
---- Python-3.8.2/Lib/site.py    2020-04-28 12:48:38.000000000 -0700
-+++ Python-3.8.2-new/Lib/site.py        2020-04-28 12:52:46.000000000 -0700
-@@ -487,7 +487,8 @@
+--- Python-3.11.4/Lib/site.py	2023-06-07 03:30:27.000000000 +0530
++++ Python-3.11.4.mod/Lib/site.py	2023-07-08 11:31:15.088734501 +0530
+@@ -523,8 +523,7 @@
                      if key == 'include-system-site-packages':
                          system_site = value.lower()
                      elif key == 'home':
 -                        sys._home = value
-+                        # this is breaking pyconfig.h path detection with venv
-+                        print('Ignoring "sys._home = value" override')
- 
+-
```

Comment:
Any reason for removing this comment?

Reply:
HI @T-Dynamos,

Any updates about this one?

Reply:
I think this is also not needed.